docker/build-tools/build-and-push.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -x
```

Comment:
Setting `set -eu` makes from my point of view in general sense to catch errors and typos in variable names.

Reply:
I agree.